sources/packages/web/src/views/institution/locations/confirmation-of-enrollment/LocationCOESummary.vue

```diff
@@ -71,55 +55,8 @@ export default {
       required: true,
     },
   },
-  setup(props: any) {
-    const router = useRouter();
-    const { dateString } = useFormatters();
-    const applications = ref([] as COESummaryDTO[]);
-
-    const goToViewApplication = (applicationId: number) => {
-      router.push({
-        name: InstitutionRoutesConst.COE_EDIT,
-        params: { locationId: props.locationId, applicationId },
-      });
-    };
-
-    const updateSummaryList = async (locationId: number) => {
-      applications.value = await ConfirmationOfEnrollmentService.shared.getCOESummary(
-        locationId,
-      );
-    };
-
-    watch(
-      () => props.locationId,
-      async currValue => {
-        //update the list
-        await updateSummaryList(currValue);
-      },
-    );
-
-    onMounted(async () => {
-      await updateSummaryList(props.locationId);
-    });
-
-    const getCOEStatusColorClass = (status: string) => {
-      switch (status) {
-        case COEStatus.completed:
-          return "bg-success text-white";
-        case COEStatus.required:
-          return "bg-warning text-white";
-        case COEStatus.declined:
-          return "bg-danger text-white";
-        default:
-          return "";
-      }
-    };
-
-    return {
-      applications,
-      dateString,
-      goToViewApplication,
-      getCOEStatusColorClass,
-    };
+  setup() {
+    return {};
   },
```

Comment:
Is there need for setup since an empty object with no attributes is returned ?

Reply:
makes sense. will remove it.

Reply:
I need to return enrollmentStatus now. so need the setup again.